Packages/io.chainsafe.web3-unity/Runtime/Common/Scripts/ILogin.cs

```diff
@@ -13,30 +13,28 @@
 using UnityEngine;
 using UnityEngine.SceneManagement;
 
-namespace Scenes
+namespace ChainSafe.Gaming.UnityPackage.Common
 {
-    public abstract class Login : MonoBehaviour
+    /// <summary>
+    /// Builds <see cref="Web3"/> Instance and Login using a Wallet or a provider.
+    /// </summary>
+    public interface ILogin
     {
-        [field:SerializeField] public string SceneToLoad { get; private set; }
-
-        public static int LoginSceneIndex { get; private set; } = 0;
-
-        [SerializeField] private string gelatoApiKey = "";
-
-        [SerializeField] private ErrorPopup errorPopup;
-
-        private IEnumerator Start()
+        public string GelatoApiKey { get; }
```

Comment:
What if I don't want to use gelato 😄

Reply:
Are we moving away from Gelato since they're not gasless anymore?

Reply:
Because we set it up like this before

Reply:
Still giving the talk regarding it on monday so i vote we leave it in until it's deprecated. May aswell get the most out of our work here.